src/main/java/com/couchbase/lite/Database.java

```diff
@@ -1225,8 +1226,17 @@ public synchronized void open(DatabaseOptions options) throws CouchbaseLiteExcep
         open = true;
 
         if (upgrade) {
-            // TODO: Perform database upgrading here:
-            // https://github.com/couchbase/couchbase-lite-java-core/issues/722
+            Log.i(TAG, "Upgrading to %s ...", storageType);
+            String dbPath = new File(path, "db.sqlite3").getAbsolutePath();
```

Comment:
Is this upgrade method support only Android/Java v1.2.0 SQLite database? How about pervious version?